contributors.json

```
@@ -29,6 +29,10 @@
     "name": "Dhondup Lama",
     "twitter": "https://twitter.com/titungdup"
   },
+   "isabellabond": {
+    "country": "United States",
+    "name": "Isabella Bond", 
```

```suggestion
    "name": "Isabella Bond" 
```

The `json` syntax doesn't allow trailing commas.